server/controllers/game.js

```diff
@@ -77,10 +77,10 @@ const start = (req, res, next) => {
   res.locals.params = [];
 
   const io = req.app.get('socketio');
-  const game = req.locals.game;
+  const game = res.locals.game;
   setInterval(() => {
     game.timerCountDown();
-    io.emit('timer', game.gameState.timer);
+    io.to(req.params.id).emit('update', game.gameState);
   }, 1000);
```

Comment:
Looks weird. It seems those changes were made in previous commits long ago. This snippet of code in current version of dev looks like: 
https://github.com/hatchways/team-fruit-loops/blob/d246de2bdc5969ca1916b2c12fbe59e06ec23df8/server/controllers/game.js#L79-L88

I am afraid that these changes would overwrite current dev.

Reply:
It also happens to other files except those changed in your recent commits: d410b50, 8afb3cc, and bc18d04.

Reply:
Perhaps you merged a previous version of skeleton branch which does not contain latest changes and updates.

Reply:
I made the branch before the others were created. I'm going to merge dev into my branch, before I merge into dev